staging/src/k8s.io/apiserver/pkg/storage/etcd3/logger.go

```diff
@@ -32,15 +32,21 @@ type klogWrapper struct{}
 const klogWrapperDepth = 4
 
 func (klogWrapper) Info(args ...interface{}) {
-	klog.InfoDepth(klogWrapperDepth, args...)
+	if klog.V(5).Enabled() {
```

Comment:
how do we decide on these levels anymore?
we run kind at 4 in CI I think, so this neatly avoids these, but I think 4 was semi-arbitrary (At best maybe copying kube-up test-config?)

Reply:
I don't have a very satisfying answer, other than V(5) is the level at which kube-apiserver starts spitting out so much debug info you must not care about log volume

Reply:
you could also use vmodule to increase verbosity just for this file if you wanted to output these without drowning in other logs